src/assets/locales/es.json

```diff
@@ -173,288 +173,288 @@
+  "autoexit.title": "Finalización automática de la sesión en ",
   "autoexit.title_units": " segundos",
-  "autoexit.concurrent_subtitle": "Has iniciado otra sesión.",
-  "autoexit.idle_subtitle": "Has estado inactivo durante demasiado tiempo.",
+  "autoexit.concurrent_subtitle": "Has empezado una nueva sesión.",
+  "autoexit.idle_subtitle": "Has estado inactivo por demasiado tiempo.",
   "autoexit.cancel": "CANCELAR",
-  "presence.entered_room": "has entrado en la habitación.",
-  "presence.entered_lobby": "has entrado en el vestíbulo.",
-  "presence.join_lobby": "te has unido al lobby.",
-  "presence.join_room": "te has unido a la sala.",
-  "presence.leave": "izquierda.",
-  "presence.name_change": "ahora se te conoce como",
-  "presence.scene_change": "has cabiado la escena a",
-  "presence.hub_name_change": "has cabiado el nombre de la habitación a",
-  "presence.in_lobby": "Vestíbulo",
-  "presence.entering": "Entrando a la habitación",
-  "presence.in_room": "En la habitación",
-  "presence.mute_all": "Silenciar todo",
-  "home.create_a_room": "Crear una habitación",
-  "home.desktop.add_pwa": "Instalar aplicación de escritorio",
-  "home.mobile.add_pwa": "Agregar a la pantalla de inicio",
-  "home.take_a_tour": "Haga un tour",
+  "presence.entered_room": "entró a la sala.",
+  "presence.entered_lobby": "entró a la antesala.",
+  "presence.join_lobby": "se unió a la antesala.",
+  "presence.join_room": "se unió a la sala.",
+  "presence.leave": "se fue.",
+  "presence.name_change": "ahora se llama",
+  "presence.scene_change": "ha cambiado la escena a",
+  "presence.hub_name_change": "ha cambiado el nombre de la sala a",
+  "presence.in_lobby": "Antesala",
+  "presence.entering": "Entrando a la sala",
+  "presence.in_room": "En la sala",
+  "presence.mute_all": "Silenciar a todos",
+  "home.create_a_room": "Crear una sala",
+  "home.desktop.add_pwa": "Instalar la aplicación para escritorio",
+  "home.mobile.add_pwa": "Agregar a la pantalla principal",
+  "home.take_a_tour": "Hacer un recorrido",
   "home.room_create_options": "opciones",
-  "home.room_create_button": "Crear habitación",
-  "home.create_name.validation_warning": "Nombre no válido, limitado a 4 a 64 caracteres y símbolos limitados.",
-  "home.powered_by_prefix": "Desarrollado por",
-  "home.powered_by_link": "Cloud Hubs",
+  "home.room_create_button": "Crear una sala",
+  "home.create_name.validation_warning": "Nombre inválido. Debe tener entre 4 a 64 caracteres y símbolos limitados.",
+  "home.powered_by_prefix": "Hecho con la tecnología de",
+  "home.powered_by_link": "Hubs Cloud",
   "home.join_us": "Únete a la conversación",
-  "home.subscribe_to_mailing_list": "Suscríbete para recibir actualizaciones",
-  "home.have_code": "¿Tienes un código de habitación?",
-  "home.add_to_discord_1": "Agregar el",
+  "home.subscribe_to_mailing_list": "Suscríbete para actualizaciones",
```

Comment:
"Suscríbete para actualizaciones" should be "Suscríbete para recibir actualizaciones"